packages/forms-web-app/src/routes/examination/about-the-application.js

```diff
@@ -3,8 +3,8 @@ const router = express.Router();
 
 const aboutTheApplicationController = require('../../controllers/examination/about-the-application');
 
-router.get('/:page', aboutTheApplicationController.getAboutTheApplication);
-router.post('/search/:page', aboutTheApplicationController.postSearchDocument);
+router.get('/:case_ref/:page', aboutTheApplicationController.getAboutTheApplication);
+router.post('/search/:case_ref/:page', aboutTheApplicationController.postSearchDocument);
```

Comment:
Don't you think it would be better to have the `:case_ref` before the `/search` portion of the URL?  That would be consistent with how we link to the examination route in the first place, e.g. `/examination/EN010118`.  Thus, the search route would follow more logically as a sub-resource: `/examination/EN010118/about-the-application/search` or maybe better still: `/examination/EN010118/search`

Reply:
Its updated to **/examination/EN010118/about-the-application/search**